backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RdfConfig.java

```diff
@@ -0,0 +1,19 @@
+package org.molgenis.emx2.rdf;
+
+import org.molgenis.emx2.rdf.generators.RdfApiGeneratorFactory;
+import org.molgenis.emx2.rdf.writers.WriterFactory;
+
+// todo: replace placeholder with full implementation that uses advaned setting to store values
+public class RdfConfig {
+  private final WriterFactory writerFactory = WriterFactory.valueOf("MODEL");
```

Comment:
what is background of this idea?

Reply:
To easily allow for this: https://github.com/molgenis/molgenis-emx2/issues/4846

The RdfConfig should load/parse the advanced setting (with YAML as value?) so it's usable elsewhere.

Reply:
Both semantic-only output and moving from a ModelBuilder to streaming would require significant changes to the code, so to make things easier and keep the code readable these changes were made. That we can now even switch between a ModelBuilder & streaming per-schema is just an added bonus.